src/traces/scattermapbox/convert.js

```diff
@@ -261,6 +263,9 @@ function makeSymbolGeoJSON(calcTrace, gd) {
             calcPt.txt = Lib.texttemplateString(txti, {}, gd._fullLayout._d3locale, calcPt, trace._meta || {});
         }
 
+        var text = txt ? calcPt.txt : fillText(calcPt.tx);
+        if(text) text = text.replace(NEWLINES, '').replace(BR_TAG_ALL, '\n');
```

Comment:
Interesting. 
Wondering this logic may be useful in other places. 
If so, it would be nice to have it in `Lib`.

Reply:
It's part of https://github.com/plotly/plotly.js/blob/4ef2f802e0c940a62cec9ef8735cd7e8972796b5/src/lib/svg_text_utils.js#L449-L626

used for all SVG text.